SparkPost/2web2ui
# AC-1944 - rename LargeForm and Form to LargeMain and Main

### What Changed

- LargeForm.js renamed to LargeMain.js
- Form.js renamed to Main.js
- Tests and imports updated

### How To Test

- Form -> Main
- LargeForm -> LargeMain

- Enable2FA- https://github.com/SparkPost/2web2ui/pull/2123/files#diff-ec169d4f5084fe885b50954b2758d16ef2c2d166b8a58b1eb6a273861648e05dR112
- RV Onboarding- https://github.com/SparkPost/2web2ui/pull/2123/files#diff-ec169d4f5084fe885b50954b2758d16ef2c2d166b8a58b1eb6a273861648e05dR614

- We need to load up those routes and make sure there's no issues or differences. It should look the same as before. (nothing was changed, file was just renamed and import was updated.)

### To Do
